tests/e2e_test/deletion_status_propagation_test.go

```diff
@@ -0,0 +1,85 @@
+package e2e_test
+
+import (
+	declarative "github.com/kyma-project/lifecycle-manager/internal/declarative/v2"
+	"k8s.io/apimachinery/pkg/api/meta"
+
+	"github.com/kyma-project/lifecycle-manager/api/v1beta2"
+	. "github.com/kyma-project/lifecycle-manager/pkg/testutils"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+const moduleName = "template-operator"
+
+var _ = Describe("Warning Status Propagation When Deletion Timestamp Not Zero", Ordered, func() {
+	kyma := NewKymaWithSyncLabel("kyma-sample", "kcp-system", "regular",
+		v1beta2.SyncStrategyLocalSecret)
+
+	InitEmptyKymaBeforeAll(kyma)
+	CleanupKymaAfterAll(kyma)
+
+	BeforeAll(func() {
+		// make sure we can list Kymas to ensure CRDs have been installed
+		err := controlPlaneClient.List(ctx, &v1beta2.KymaList{})
+		Expect(meta.IsNoMatchError(err)).To(BeFalse())
+	})
```

Comment:
we don't need this beforeall, not necessary, you can remove this part in other tasks as a refactoring purpose.